backend/faq/views.py

```diff
@@ -98,15 +98,14 @@ def get(self, request):
         try:
             high_level_instance = HighLevelModel.objects.get(user_id=request.user.id)
             faq_data = list(FAQ.objects.filter(high_level_id=high_level_instance.id).values("question", "answer"))
-            download_csv_file(faq_data)
-            return Response(
-                {    
-                    "details": "download csv file",
-                    "success": True
-                },
-                status=status.HTTP_200_OK)
-        except Exception:
-            return Response("Please integrate your high level Integrations")
+            df = pd.DataFrame(faq_data)
+            csv_data = df.to_csv(index=False)
+            response = HttpResponse(csv_data, content_type='text/csv')
+            response['Content-Disposition'] = 'attachment; filename="my_file.csv"'
+            return response
+        except Exception as e:
+            print("your eroor is ", e)
+            return response(str(e))
```

Comment:
Don't send your exception errors in response, You can just send Download Failed.
And It should be Response not response